spec/models/miq_group_spec.rb

```diff
@@ -326,11 +326,15 @@
     let(:group) { FactoryBot.create(:miq_group) }
 
     it "uses dashboard_order if present" do
+      # clear settings out of the group so we don't get surprised in the test
+      group.settings = nil
+
       ws1 = FactoryBot.create(:miq_widget_set, :name => 'A1', :owner => group)
       FactoryBot.create(:miq_widget_set, :name => 'C3', :owner => group)
       ws3 = FactoryBot.create(:miq_widget_set, :name => 'B2', :owner => group)
-
-      group.update(:settings => {:dashboard_order => [ws3.id.to_s, ws1.id.to_s]})
+      group.add_to_dashboard_order(ws3.id)
+      group.add_to_dashboard_order(ws1.id)
```

Comment:
I feel more comfortable when we use the api instead of mucking with blobs. But in the end, `settings` will have the same value. I also wanted to ensure that `add_to_dashboard_order` added them in the correct order.